Ginger/Ginger/UserControlsLib/ChatBot/ChatbotWindow.xaml.cs

```diff
@@ -70,9 +69,9 @@ private async Task SendMessageToAPI()
             ShowLoader();
             try
             {
-                //need to handle as we are getting token everytime
 
-                
+                genAIServiceHelper = new GenAIServiceHelper();
```

Comment:
its not recommended to create this each time user sends a message. please initiate it once, and re-initialize only when its null